### ISSUE TITLE
Update cmd + f to open and focus input & esc to close input #Suggestion

### ISSUE THREAD

**Issue by ldexterldesign** [jaanauati/hyper-search#74]
Hi,

Hope you're well

Thanks for software

As title

# Steps to reproduce issue

n/a

# Unexpected behaviour

Currently, `cmd` + `f` toggles (i.e. open/close) input

# Expected behaviour

`cmd` + `f` to open and focus input
`esc` to close input

This is how everything else works in macOS (e.g. Finder, Vivaldi, VSCodium) - please utilise this convention

If you're reading and:

* agree, then please vote up (e.g. +1, like), or;
* disagree, then please vote down (e.g. -1, dislike) - I'd love to know why?

If documentation exists (to configure this) then please provide URL or else consider creating some

If you have any issues (e.g. questions/queries) then happy to help

Yours hopefully

hardware: Apple, MacBook Pro (16-inch, 2019)
software - operating system: Apple, macOS, 10.15.3 (19D76)
software - application: Hyper 3.0.2 (stable)

**Comment by ldexterldesign** [comment]
In addition how/where can I configure the search direction?

By default search terms are highlighted bottom to top or down to up. I'd like to reverse this, like how every  other piece of software I use works (i.e. top to bottom, left to right)

Yours hopefully

**Comment by jaanauati** [comment]
hi @ldexterldesign!, hope u are doing well!,
it's not clear for me what the issue is, ... is the problem that the search dialog is not showing up when pressing cmd+f?.
thanks

**Comment by ldexterldesign** [comment]
No, the search is showing up, it's just the interaction I suggest you improve

Possibly a language/translation issue because not sure how much clearer I can be

The important part of my request...

Currently, `cmd` + `f` toggles (i.e. opens/closes) the search input but I suggest instead that `cmd` + `f` opens the input and `esc` closes the input

Hope this is clear(er) and to hear back

Thanks

**Comment by ldexterldesign** [comment]
BUMP
